### DISCUSSION TITLE
typecho的评论数据导入到leancloud，报错

### DISCUSSION THREAD

**Discussion by zqlit** [walinejs/waline#1202] [Q&A] [answered]
在typecho后台装 Export2Valine 插件将评论数据导出成 Valine 数据
然后在Class 名称输入“Comment”，数据文件选择刚才导出的 JSON 文件
然后提示这个报错，请问作者该如何解决呢？
![158e99792ed69c3f051eb1f9eeb5bc1.png](https://img.usj.cc/i/2022/07/17/62d3fe3b5a97d.png)

**Comment by lizheming** [comment, marked as answer]
https://github.com/lizheming/typecho-export-valine/commit/af0e42188887d8e71a43af611659b2c5513965d1

更新了一版导出格式，可以再试试，如果还不行可以把导出的文件发我一下试试，我的邮箱个人资料上有。
